src/modals/Deploy.tsx

```diff
@@ -83,43 +79,42 @@ function Deploy() {
 
         console.log(`Your receipt is`);
         console.log(receipt);
-        return receipt.status;
+        return true;
```

Comment:
Why do we always return true now instead of returning `receipt.status`?

Reply:
updated